### ISSUE TITLE
WKBWriter 3-dimensional output does not work with SpatiaLite

### ISSUE THREAD

**Issue by hrstka** [locationtech/jts#710]
Setting the high bit of the wkbType word in case of 3D geometry does not work with SpatiaLite. According to [wikipedia](https://en.wikipedia.org/wiki/Well-known_text_representation_of_geometry) the presence of 3D coordinates should be indicated by different geometry type code (e.g. 2D point is 1 and 3D point is 1001).

**Comment by dr-jts** [comment]
Yes, the `WKBWriter` is still outputting the old PostGIS EWKB format, rather than the current ISO/OGC standard WKB format.  (Note that the `WKBReader` supports both formats).

It seems like the way forward is to enhance the `WKBWriter` to output ISO/OGC standard, with the old format left as an option.
